tests/book.test.js

```diff
@@ -0,0 +1,209 @@
+            author: 'Homer'
+          });
+
+          const testAuthorThree = Author.create({
+            author: 'George Orwell'
+          }); 
+
+          const testGenreOne = Genre.create({
+            genre: 'Romance'
+          });
+
+          const testGenreTwo = Genre.create({
+            genre: 'Epic poetry'
+          });
+
+          const testGenreThree = Genre.create({
+            genre: 'Dystopian'
+          });
+
+        books = await Promise.all([
+          Book.create({
+            title: 'Pride and prejudice',
+            author: testAuthorOne.id,
+            genre: testGenreOne.id,
+            ISBN: '9780140430721',
+          }),
+          Book.create({ 
+            title: 'The Illiad', 
+            author: testAuthorTwo.id,
+            genre: testGenreTwo.id,
+            ISBN: '9780800042011'
+          }),
+          Book.create({
+            title: 'Nineteen Eighty-Four',
+            author: testAuthorThree.id,
+            genre: testGenreThree.id,
+            ISBN: '9780140817744'
+          }),
+        ]);
+    });   
+
+    describe('GET /books', () => {
+        it('gets all books records', async () => {
+            const response = await request(app).get('/books');
+
+            expect(response.status).to.equal(200);
+            expect(response.body.length).to.equal(3);
+
+            response.body.forEach((book) => {
+                const expected = books.find((a) => a.id === book.id);
+
+                expect(book.title).to.equal(expected.title);
+                expect(book.ISBN).to.equal(expected.ISBN);
+            });
+        });
+    });
+
+    describe('GET /books/:id', () => {
+        it('gets book records by id', async () => {
+            const book = books[0];
+            const response = await request(app).get(`/books/${book.id}`);
+
+            expect(response.status).to.equal(200);
+            expect(response.body.title).to.equal(book.title);
+        });
+
+        it('returns a 404 if the book does not exist', async () => {
+            const response = await request(app).get('/books/12345');
+
+            expect(response.status).to.equal(404);
+            expect(response.body.error).to.equal('The book could not be found.');
+        });
+    });
+
+    describe('PATCH /books/:id', () => {
+        it('updates book by id', async () => {
+            const book = books[0];
+            const response = await request(app)
+            .patch(`/books/${book.id}`)
+            .send({ title: 'Pride and Prejudice'});
+
+            const updateBookRecord = await Book.findByPk(book.id, {
+                    raw: true
+            });
+
+            expect(response.status).to.equal(200);
+            expect(updateBookRecord.title).to.equal('Pride and Prejudice');
+        });
+
+        it('returns a 404 if book does not exist', async () => {
+            const response = await request(app)
+            .patch('/books/12345')
+            .send({ title: 'some book'});
```

Comment:
Chained calls should be indented
```suggestion
                .patch('/books/12345')
                .send({ title: 'some book'});
```